pkgs/development/libraries/linbox/default.nix

```diff
@@ -36,6 +35,16 @@ stdenv.mkDerivation rec {
     fflas-ffpack
   ];
 
+  patches = [
+    ./patches/linbox-pr256-part2.patch # TODO: Remove on 1.7.0 update
+  ];
+
+  postPatch = ''
+    # Part of https://github.com/linbox-team/linbox/pull/249
+    # TODO: Remove on 1.7.0 update
```

Comment:
Please add an additional comment to the `version` attribute, since its less likely to be missed there when updating. That is especially important for a `sed` script, since it won't fail the build if it doesn't apply anymore.

```
  version = "1.6.3"; # TODO remove the postPatch script on update
```

Reply:
OK.